.github/workflows/go-test.yml

```diff
@@ -4,17 +4,32 @@ on:
   workflow_call:
     inputs:
       try-build:
-        description: 'Attempt to build the project'
+        description: "Attempt to build the project"
         type: boolean
         required: false
         default: true
+      go-test-args:
+        description: "Arguments to pass to `go test`"
+        type: string
+        required: false
+        default: -race;-tags=netgo
+      go-test-package:
+        description: "Package to pass to `go test`"
+        type: string
+        required: false
+        default: ./...
+      go-test-setup:
+        description: Enables/Disables the optional test setup step
+        type: boolean
+        required: false
+        default: false
 
 jobs:
   test:
     strategy:
       matrix:
-        go-version: [ 1.22, 1.23 ]
-        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        go-version: [1.22, 1.23]
```

Comment:
Why do you hate `[ foo, bar ]`? 🤣

Reply:
It's not me. It's the IDE's auto-format. Embrace auto formatting 😅